src/layout/IHyprLayout.cpp

```diff
@@ -9,21 +9,28 @@
 #include "../xwayland/XSurface.hpp"
 
 void IHyprLayout::onWindowCreated(PHLWINDOW pWindow, eDirection direction) {
-    if (pWindow->m_bIsFloating) {
-        onWindowCreatedFloating(pWindow);
+    CBox desiredGeometry = {};
+    g_pXWaylandManager->getGeometryForWindow(pWindow, &desiredGeometry);
+
+    if (desiredGeometry.width <= 5 || desiredGeometry.height <= 5) {
+        const auto PMONITOR          = g_pCompositor->getMonitorFromID(pWindow->m_iMonitorID);
+        pWindow->m_vLastFloatingSize = PMONITOR->vecSize / 2.f;
     } else {
```

Comment:
my friend

Reply:
this fixes a bug where the window has 0 size when:
+ allfloat = true
+ create a new window, tile it, make it a group, and create a new window in the tiled group.
+ togglefloat and the window disappears.